src/Cake.Common/Tools/ILRepack/ILRepackRunner.cs

```diff
@@ -127,7 +127,7 @@ protected override IEnumerable<string> GetToolExecutableNames()
 
             if (settings.TargetPlatform.HasValue)
             {
-                builder.Append(GetTargetPlatformString(settings.TargetPlatform.Value));
+                builder.Append("/targetplatform:" + GetTargetPlatformString(settings.TargetPlatform.Value));
```

Comment:
I would prefer if we used `builder.AppendSwitch("/targetplatform:", GetTargetPlatformString(settings.TargetPlatform.Value))` to be more in line with other tools.